app/src/main/java/com/github/geohunt/app/model/database/firebase/FirebaseChallenge.kt

```diff
@@ -23,6 +22,8 @@ data class FirebaseChallenge(
     override val expirationDate: LocalDateTime?,
     override val correctLocation: Location,
     override val claims: List<LazyRef<Claim>>,
+    override var likes: List<LazyRef<User>>,
+    override var nbLikes: Int
```

Comment:
Same as above

Reply:
I have refactored the name of the variable to numberOfLikes